### ISSUE TITLE
Define decoding method for loadContents and cwl.output.json

### ISSUE THREAD

**Issue by ThomasHickman** [common-workflow-language/common-workflow-language#736]
> If the output directory contains a file named "cwl.output.json", that file must be loaded and used as the output object
> Read up to the first 64 KiB of text from the file and place it in the "contents" field of the file object for use by expressions.

There are two locations in the CWL specification which specify that an implementation must read a file and interpret as text. It it not clear what encoding an implementation should use when decoding these files.

**Comment by tetron** [comment]
Yes, I think it should specify UTF-8, but you are correct that it isn't explicit.
